metricflow/test/api/conftest.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from metricflow.api.simple_api import MetricFlowSimpleAPI
+from metricflow.model.semantic_model import SemanticModel
+from metricflow.plan_conversion.time_spine import TimeSpineSource
+from metricflow.protocols.sql_client import SqlClient
+from metricflow.test.fixtures.setup_fixtures import MetricFlowTestSessionState
+
+
+@pytest.fixture
+def mf_simple_api(
+    create_simple_model_tables: bool,
+    sql_client: SqlClient,
+    simple_semantic_model: SemanticModel,
+    time_spine_source: TimeSpineSource,
+    mf_test_session_state: MetricFlowTestSessionState,
+) -> MetricFlowSimpleAPI:
+    """Fixture for MetricFlowSimpleAPI."""
+    return MetricFlowSimpleAPI(
+        sql_client=sql_client,
+        semantic_model=simple_semantic_model,
+        system_schema=mf_test_session_state.mf_system_schema,
```

Comment:
Is it possible to have the fixture set up so that there is a test config in place and the config handler simply reads it in? That'd make this a bit better as a test of the client class mechanics and prevent us from exposing these parameter overrides.

Reply:
For now, it doesn't seem possible since the config only supports building via redshift/snowflake/bigquery and since we use sqlite for the testing this will be what we have to do in the meantime (unless we pass real creds via github secrets?).

Reply:
I think this is an issue but it's outside the scope of this PR. We want a way to shim the config in tests so anything that does automatic access via the config object can be tested in its own right without us wiring in fully-formed instances as overrides like this.

For testing non-SQLite engines we could have something that reads warehouse credentials out of the environment (as we do today), so I don't think that's a huge blocker. The idea is basically to push the fixture work that initializes the SqlEngine etc. one level further out so it generates a config object.